src/components/send/SendContainerV2.js

```diff
@@ -0,0 +1,209 @@
+import BN from 'bn.js';
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+
+import isDecimalString from '../../utils/isDecimalString';
+import Container from '../common/styled/Container.css';
+import EnterAmount from './components/views/EnterAmount';
+import EnterReceiver from './components/views/EnterReceiver';
+import Review from './components/views/Review';
+import SelectToken from './components/views/SelectToken';
+
+//TODO: Handle min-height when showing top banner
+const StyledContainer = styled(Container)`
+    &&& {
+        .header {
+            position: relative;
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            color: #272729;
+            font-weight: 600;
+            font-size: 20px;
+    
+            .back-arrow-button {
+                position: absolute;
+                left: 0;
+            }
+        }
+
+        .buttons-bottom-buttons {
+            margin-top: 55px;
+    
+            > button {
+                display: block;
+                width: 100%;
+            }
+    
+            .link {
+                display: block;
+                margin: 20px auto;
+            }
+        }
+    
+        @media (max-width: 500px) {
+            .buttons-bottom {
+                display: flex;
+                flex-direction: column;
+        
+                .buttons-bottom-buttons {
+                    margin-top: auto;
+                }
+        
+                &.enter-amount {
+                    min-height: calc(100vh - 110px);
+                }
+            }
+        }
+    }
+`;
+
+const SendContainerV2 = ({
+    FTMethods,
+    availableNearBalance,
+    reservedNearForFees,
+    availableNearToSend,
+    redirectTo,
+    fungibleTokens,
+    checkAccountAvailable,
+    localAlert,
+    clearLocalAlert,
+    accountId
+}) => {
+
+    const [amount, setAmount] = useState('');
+    const [estimatedTotalFees, setEstimatedTotalFees] = useState('0');
+    const [estimatedTotalInNear, setEstimatedTotalInNear] = useState('0');
+    const [parsedAmount, setParsedAmount] = useState('');
+    const [receiverId, setReceiverId] = useState('');
+    const [view, setView] = useState('enterAmount');
+    const [maxAmount, setMaxAmount] = useState(null);
+    const [sendingToken, setSendingToken] = useState(false);
+    const [selectedToken, setSelectedToken] = useState(fungibleTokens[0]);
+
+    useEffect(() => {
+        selectedToken.symbol === 'NEAR' && setSelectedToken(fungibleTokens[0]);
+    }, [availableNearToSend]);
```

Comment:
Since the calculated `availableNearToSend` isn't present when the component first mounts, we still need to listen for change to `availableNearToSend` and then `setSelectedToken(fungibleTokens[0])` once it's finished calculating the balance.